fahclient/values.yaml

```
@@ -70,10 +74,27 @@ securityContext:
       - ALL
   readOnlyRootFilesystem: false
 
+# Don't forget to enable external access in configmap, to access service
 service:
   type: ClusterIP
   port: 80
 
+# Ingress for WebUI
+# Don't forget to enable external access in configmap, too
+ingress:
+  enabled: false
+  hostname: chart-example.local
+  path: /
+
+  # Enable TLS for Ingress e.g. letsencrypt using cert-manager
+  tls: false
+
+  # Ingress Annotations
+  annotations:
```

```suggestion
  annotations: {}
```
IIRC, the chart will fail to deploy because the value isn't provided here for annotations.